R/playername_lookup.R

```diff
@@ -0,0 +1,24 @@
+#' Look up Baseball Player Name
+#'
+#' This function allows you to query the Chadwick Bureau's public register of baseball players and the various IDs associated with them in different systems of record.
+#' @param id An integer or character string representing a player ID in a baseball database, cross-referenced through the Chadwick Bureau's public register of baseball players.
+#' @keywords MLB, sabermetrics
+#' @export
+#' @examples
+#' \dontrun{playername_lookup(4885)}
+#' \dontrun{playername_lookup("kaaihki01")}
+
+playername_lookup <- function(id) {
+  if (!exists("chadwick_player_lu_table")) {
+    print("Be patient, this may take a few seconds...")
+    print("Data courtesy of the Chadwick Bureau Register (https://github.com/chadwickbureau/register)")
+    url <- "https://raw.githubusercontent.com/chadwickbureau/register/master/data/people.csv"
+    chadwick_player_lu_table <- read.csv(url)
```

Comment:
Can you switch this to `readr::read_csv()`? That will allow for the initial download of the Chadwick table to be quicker. I need to make this same change to the `playerid_lookup` function.

Reply:
You saw that this change was made in #75 right?

Reply:
I missed it, but see it now.

Reply:
Good catch-- agree with moving from read.csv to read_csv, I was just lazily copying from another function. Will change